modules/psap-special-resource-operator.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="about-special-resource-operator_{context}"]
+= About the Special Resource Operator
+
+The Special Resource Operator (SRO) helps you manage the deployment of kernel modules and drivers on an existing {product-title} cluster. The SRO can be used for a case as simple as building and loading a single kernel module, or as complex as deploying the driver, device plugin, and monitoring stack for a hardware accelerator.
+
+For loading kernel modules, SRO is designed around the use of driver containers. Driver containers are increasingly being used in cloud-native environments, especially when run on pure container operating systems, to deliver hardware drivers to the host. Driver containers extend the kernel stack beyond the out-of-the-box software and hardware features of a specific kernel. Driver containers work on various container-capable Linux distributions. With driver containers the host operating system stays "clean" and there will not be any clash between different library versions or binaries on the host.
```

Comment:
* Comma after "With driver containers" ?
* Can we removed the quotes around clean? If we don't like the word clean enough alone without the quotes, then maybe another word could be used, or the sentence reworded to avoid having to use it.
* "and there will not be any" -> "there is no" or something else to avoid the "will"?